src/apps/geonetwork/src/main/java/org/geonetwork/gn4proxypredicates/CachingPortalIds.java

```diff
@@ -24,8 +24,9 @@
  * request for the portal uuids.
  */
 public class CachingPortalIds {
-    static ApplicationContext applicationContext = ApplicationContextProvider.getApplicationContext();
     static LoadingCache<String, List<String>> cache;
+    // FIXME: is ok in test, but null at runtime
+    static ApplicationContext applicationContext = ApplicationContextProvider.getApplicationContext();
```

Comment:
not sure what you mean - this shouldn't be null at runtime.  In my version, its not null...